### PR TITLE
Add freeze_time to verifier test which gives nondeterministic results

### PR DESCRIPTION
**Pull request by kbeker**
Add freeze_time because this test gives sometimes different time at the end of it. `freeze_time()` resolves it

### CODE REVIEW

File: concent_api/verifier/tests/test_integration_verification.py [hunk 1]
```diff
@@ -370,44 +371,45 @@ def test_blender_verification_order_should_call_verification_result_with_result_
         )
 
     def test_that_blender_verification_order_should_work_properly_when_there_is_more_than_one_frame_to_render(self):
-        with mock.patch('verifier.tasks.download_archives_from_storage', autospec=True) as mock_download_archives_from_storage,\
```
Comment (cameel) [inline concent_api/verifier/tests/test_integration_verification.py]: When doing something non obvious like this you should always remember to add a comment explaining why you're doing it.

Also, say which part is non-deterministic and fails.

Comment (kbeker) [reply concent_api/verifier/tests/test_integration_verification.py]: Should I comment this inside test?

Comment (kbeker) [reply concent_api/verifier/tests/test_integration_verification.py]: Updated.

Comment (rwrzesien) [reply concent_api/verifier/tests/test_integration_verification.py]: We should put more comments in code in general :)

Comment (cameel) [reply concent_api/verifier/tests/test_integration_verification.py]: > Should I comment this inside test?

Yes.

> We should put more comments in code in general :)

Agreed. But please avoid comments that just say what you're doing. The comments should explain why and give context that's not available in the code they describe.

**Pull request merged**